index.html

```diff
@@ -9,45 +9,217 @@
+                    </ul>
+                </div>
+                <button class="resume">See Project</button>
+            </div>
+            
+          </div>
+          <!-- end -->
+          <!-- Multi-Post Stories -->
+        <div class="grid-container">
+            <div class="grid-item"><img src="images/multi-post-stories.jpg" alt="works 1"></div>
+            <div class="grid-item">
+                <div class="works1">
+                    <h1 class="salutation">Multi-Post
+                        Stories</h1>
+                    <div class="flexf">
+                        <p class="canopy">canopy</p> 
+                        <img src="images/Counter.jpg" alt="counter">
+                        <p class="back">back end dev</p>
+                        <img src="images/Counter.jpg" alt="counter">
+                        <p class="back">2015</p>
+                    </div>
+                </div>
+                <div>
+                    <p class="developer">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+              
+                </div>
+                <div>
+                    <ul>
+                        <li><button class="btn1">HTML</button></li>
+                        <li><button class="btn1">CSS</button></li>
+                        <li><button class="btn1">Javascript</button></li>
+                    </ul>
+                </div>
+                <button class="resume">See Project</button>
+            </div>
+            
+          </div>
+          <!-- end -->
+    </section>
+
+    <!-- This section gives my background -->
+
+<section class="about-me">
+    <p class="salutation">About Myself </p>
+    <p class="developer">Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+
+    <p class="connect">LET'S CONNECT</p>
+
+    
+    <ul>
         <li><a href="https://twitter.com/home"><img src="images/twitter.jpg" alt="twitter"></a></li>
         <li> <a href="https://www.linkedin.com/feed/"><img src="images/Linkedin-icon.jpg" alt="Linkedin"></a></li>
         <li><a href="https://twitter.com/home"><img src="images/Vector.jpg" alt="twitter"></a></li>
-        <li><a href="https://github.com/ausiemass/portfolio"><img src="images/Vector (1).jpg" alt="GITHUB"></a></li>
+        <li><a href="https://github.com/ausiemass/portfolio"><img src="images/github.jpg" alt="GITHUB"></a></li>
         <li><a href="https://twitter.com/home"><img src="images/Vector (2).jpg" alt="twitter"></a></li>
-    </ul>
+    </ul> 
+
+            
+    <button class="resume">Get my resume</button>
+    <section class="skills">
+
+    <div class="lan">
+        <p class="skill">Languages</p>
+        <img src="images/arrowDown.jpg" alt="downArrow">
+    </div>
     
-</section>
```

Comment:
- [ ] Your About section looks great. However, there are a few style adjustments that need to be made and you'll be on your way to the next milestone :wink: 
- [ ] Please ensure that the spacing around your design is similar to that of the original design :+1: 
- [ ] Lastly, please don't forget to style your `Get My Resume` button properly as seen in the design, and ensure that it has all the button interaction styles I mentioned in my earlier comment :+1: 

| Your Design | The Figma Design |
|--------|--------|
| ![image](https://github.com/ausiemass/portfolio-austin/assets/102699958/b5c48b2d-a5b6-443a-92c6-fc3a2dc775a9)| ![image](https://github.com/ausiemass/portfolio-austin/assets/102699958/7e182224-255d-4a60-af5f-975295a19723) |